alf/algorithms/sac_algorithm.py

```diff
@@ -380,3 +380,6 @@ def _calc_critic_loss(self, training_info):
 
         critic_loss = critic_loss1.loss + critic_loss2.loss
         return LossInfo(loss=critic_loss, extra=critic_loss)
+
+    def _trainable_attributes_to_ignore(self):
+        return ['target_critic_network1', 'target_critic_network2']
```

Comment:
should be ['_target_critic_network1", '_target_critic_network2']

Reply:
fixed